ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/CipherSuitesJvm.kt

```diff
@@ -12,3 +12,6 @@ internal actual fun CipherSuite.isSupported(): Boolean = when (platformVersion.m
     "1.6.0" -> platformVersion.minor >= 181 || keyStrength <= 128
     else -> true
 }
+
+public actual typealias TLSException = javax.net.ssl.SSLException
```

Comment:
This is a source-compatible, but binary-incompatible change for JVM. `TLSException` has been made public in 3.0.0-beta2, so it shouldn't be extensively used yet, but I can make this change binary compatible by introducing a new class `TlsException` and deprecating the old one:

```kotlin
public actual typealias = TlsException = javax.net.ssl.SSLException

@Deprecated("Use TlsException instead")
class TLSException(message: String, cause: Throwable?) : TlsException(message, cause)
```

Reply:
Let's make it binary compatible